scripts/src/chartprreview/chartprreview.py

```diff
@@ -238,17 +250,28 @@ def check_report_success(directory, report_path, version):
         msg = f"[ERROR] Error analysing the report: {err}"
         write_error_log(directory, msg)
 
+    labels = get_labels(api_url)
+    label_names = [l["name"] for l in labels]
+
     failed = report["failed"]
     passed = report["passed"]
     if failed > 0:
-        msgs = []
-        msgs.append("[ERROR] Chart verifier report includes failures:")
-        msgs.append(f"- Number of checks passed: {passed}")
-        msgs.append(f"- Number of checks failed: {failed}")
-        msgs.append(f"- Error message(s):")
-        for m in report["message"]:
-            msgs.append(f"  - {m}")
-        write_error_log(directory, *msgs)
+        if vendor_type == "redhat":
+            print(f"::set-output name=redhat_to_community::True")
+        if "force-publish" not in label_names and vendor_type != "redhat":
+            msgs = []
+            msgs.append("[ERROR] Chart verifier report includes failures:")
+            msgs.append(f"- Number of checks passed: {passed}")
+            msgs.append(f"- Number of checks failed: {failed}")
+            msgs.append(f"- Error message(s):")
+            for m in report["message"]:
+                msgs.append(f"  - {m}")
+            write_error_log(directory, *msgs)
```

Comment:
maybe keep all the messages outside this if? It is good data to have even in the event that we force-publish

Reply:
yea better not to silence the error, will update